packages/SwingSet/test/test-devices.js

```diff
@@ -434,7 +434,7 @@ async function testCommandDeliver(t, withSES) {
   t.end();
 }
 
-test('command deliver without SES', async t => {
+test.only('command deliver without SES', async t => {
```

Comment:
This `.only` needs to be removed, else none of the other swingset tests will run.

Reply:
Gotcha.  My bad.